### DISCUSSION TITLE
How do I get the fill color of graphics

### DISCUSSION THREAD

**Discussion by lykl** [pixijs/pixijs#9072] [Q&A] [answered]
I get the color under `fill` of graphics, and it always returns a number representing white

**Comment by SuperSodaSea** [comment, marked as answer]
<https://pixiplayground.com/#/edit/vpfQYi5IlZmC0RzlOxovF>

```js
const graphics = new PIXI.Graphics();
graphics.beginFill(0x123456);
console.log(graphics.fill.color.toString(16)); // 123456
graphics.endFill(); // `fill` will reset here
console.log(graphics.fill.color.toString(16)); // ffffff
```

**Reply by lykl** [reply]
I understand. `endFill` will reset the color to `white`.
But if I didn't set the color at the beginning, it would return white, so how do I know if it's not set or white?

**Reply by SuperSodaSea** [reply]
Try `graphics.fill.visible`?

**Reply by lykl** [reply]
Oh, that's right
